<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-O0SE6H4" failures="1" tests="1" name="validateTitle" time="14.318" errors="0" timestamp="2021-03-27T10:09:45 CET">
  <testcase classname="sk.ivanmolcan.ValidateTitle" name="validateTitleApp" time="1.143">
    <failure type="java.lang.AssertionError" message="expected [featured courses111] but found [featured courses]">
      <![CDATA[java.lang.AssertionError: expected [featured courses111] but found [featured courses]
at sk.ivanmolcan.ValidateTitle.validateTitleApp(ValidateTitle.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- validateTitleApp -->
</testsuite> <!-- validateTitle -->
